src/background/stores/tab-context-store-hub.ts

```diff
@@ -177,6 +179,16 @@ export class TabContextStoreHub implements StoreHub {
             persistStoreData,
         );
         this.needsReviewCardSelectionStore.initialize();
+
+        this.assessmentCardSelectionStore = new AssessmentCardSelectionStore(
```

Comment:
I'm not opposed to the idea but are we sure the assessment card selection store operating at the tab context would work? The assessment stores operate within the global context is why I ask.

Reply:
It looks like @lisli1 included the tabID in the assessment card selection store constructor, but I would be fine with removing that and making this global. @lisli1 do you have any thoughts?

Reply:
We can remove it since it isn't used